### ISSUE TITLE
Can't reply to OP after choosing media

### ISSUE THREAD

**Issue by floppyD** [Adamantcheese/Kuroba#90]
You can't scroll up, the post form is over it.
![Screenshot_20190613-090029](https://user-images.githubusercontent.com/29208110/59430942-017e5200-8dba-11e9-903f-b2c1839bff82.png)

**Comment by Adamantcheese** [comment]
Reply drafts are saved on a per-thread basis, you can always close the reply window to quote whoever you need and it will reopen with the quoted post number or text appended to the end of your draft. Won't fix, works exactly the same like it did in Clover.

**Comment by ownBard** [comment]
I think this may be a worthwhile change. In fact, this functionality is already implemented though only after closing and re-opening the post window with the image attached.

**https://files.catbox.moe/pyp9bt.mp4**

**Comment by ownBard** [comment]
It may not be very important but it definitely would be nice to have.

**Comment by Adamantcheese** [comment]
I might lump this in with other UI updates, but it's super low priority.
